includes/layouts/settings.php

```diff
@@ -154,6 +165,47 @@ class="<?php echo $_tab === $active_tab ? esc_attr( 'active' ) : ''; ?>"><?php e
 										<div class="help-text pt-8"><?php esc_html_e( 'This setting will be used to inherit the current theme style instead of the default style. If disabled, it will be considered the individual widget/block/shortcode setting.', 'feedzy-rss-feeds' ); ?></div>
 									</div>
 								</div>
+								<div class="form-block <?php echo esc_attr( apply_filters( 'feedzy_upsell_class', '' ) ); ?>">
+									<?php echo wp_kses_post( apply_filters( 'feedzy_upsell_content', '', 'auto-categories', 'settings' ) ); ?>
+									<div class="fz-form-group">
+										<label class="form-label"><?php esc_html_e( 'Auto Categories Mapping', 'feedzy-rss-feeds' ); ?></label>
+										<table class="fz-auto-cat">
+											<tbody>
+												<?php foreach ( $mapped_categories as $index => $category_mapping ) : ?>
+												<tr>
+													<td class="fz-auto-cat-col-8">
+														<input type="text" name="auto-categories[<?php echo esc_attr( $index ); ?>][keywords]" class="form-control" placeholder="<?php esc_attr_e( 'Values separated by commas', 'feedzy-rss-feeds' ); ?>" value="<?php echo esc_attr( $category_mapping['keywords'] ); ?>"/>
+													</td>
+													<td class="fz-auto-cat-col-4">
+														<select name="auto-categories[<?php echo esc_attr( $index ); ?>][category]" class="form-control fz-select-control">
+															<option value=""><?php esc_html_e( 'Select a category', 'feedzy-rss-feeds' ); ?></option>
+															<?php
+															foreach ( $categories as $category ) {
+																$selected = $category->term_id == $category_mapping['category'] ? 'selected' : '';
+																echo '<option value="' . esc_attr( $category->term_id ) . '" ' . esc_attr( $selected ) . '>' . esc_html( $category->name ) . '</option>';
+															}
+															?>
+														</select>
+														<button type="button" class="btn btn-outline-primary<?php echo $index === 0 ? ' disabled' : ''; ?>" <?php echo $index === 0 ? 'disabled' : ''; ?>><?php esc_html_e( 'Delete', 'feedzy-rss-feeds' ); ?></button>
+													</td>
+												</tr>
+												<?php endforeach; ?>
+											</tbody>
+										</table>
+										<div class="fz-auto-cat-actions">
+											<button type="button"class="btn btn-outline-primary"><?php esc_html_e( 'Add New', 'feedzy-rss-feeds' ); ?></button>
+										</div>
+										<div class="help-text pt-8">
+											<?php
+											printf(
+												// translators: %s is a placeholder for the auto categories tag.
```

Comment:
One tip here is to include the actual value in the translator's comment to give them a better context. You can append it like this `(like "[#auto_categories]")`.